types/ember/test/ember-module-tests.ts

```diff
@@ -0,0 +1,280 @@
+import Ember from 'ember';
+import EngineInstance from '@ember/engine/instance';
+import Component from '@ember/component';
+import EmberObject from '@ember/object';
+import ComputedProperty from '@ember/object/computed';
+import Evented from '@ember/object/evented';
+// $
+Ember.$; // $ExpectType JQueryStatic
+// A
+Ember.A(); // $ExpectType NativeArray<{}>
+Ember.A([1, 2]); // $ExpectType NativeArray<number>
+// addListener
+Ember.addListener({ a: 'foo' }, 'a', {}, () => {});
+Ember.addListener({ a: 'foo' }, 'a', null, () => {});
+// addObserver
+Ember.addObserver({ a: 'foo' }, 'a', null, () => {});
+Ember.addObserver({ a: 'foo' }, 'a', {}, () => {});
```

Comment:
add test for `addObserver` with target being a Function 
and for method being a string

Reply:
We already have a wide range of tests that cover `addObserver` in depth.

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/fdb583ed26aac613aad220d788101cc4c3855d35/types/ember/test/observable.ts#L7-L24

These tests are designed to ensure we don't accidentally lose things from the `import Ember from 'ember';` import